media_mpx.services.yml

```diff
@@ -66,10 +66,25 @@ services:
     class: Drupal\media_mpx\ClientFactory
     arguments: ['@media_mpx.http_handler_stack', '@http_client_factory', '@cache.media_mpx_http']
 
+  # Defines an array or memory cache that supports a limit on the number of
+  # items.
+  cache.backend.media_mpx.array_cache_pool:
+    class: Drupal\media_mpx\ArrayCachePoolFactory
+    arguments: ['@datetime.time', '@cache_tags.invalidator.checksum', 100]
+
+  # Use the above array cache on top of the normal HTTP request cache. We use
+  # the ChainedFastBackend cache so any invalidations on the consistent cache
+  # are shared with all threads.
+  cache.backend.media_mpx.chainedfast_http:
+    class: Drupal\Core\Cache\ChainedFastBackendFactory
+    arguments: ['@settings', null, cache.backend.media_mpx.array_cache_pool]
```

Comment:
Here again, can we surround in single quotes and prefix with `@` for consistency?